<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>You win!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="win_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>160</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Return to Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="win_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>420</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Congratulation! You win the game!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="win_image">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>20</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
